roles/dtc/common/tasks/common/ndfc_interface_all.yml

```diff
@@ -59,7 +59,7 @@
 - name: Set interface_all Var
   ansible.builtin.set_fact:
     interface_all: "{{
-      interface_breakout +
+      interface_breakout_remove +
```

Comment:
Why are we changing this from interface_breakout to interface_breakout_remove?  Not sure I follow this